### PR TITLE
意见箱  提案库显示更多信息

### LINKED ISSUE #329: 增加“意见箱”页面
### Is there an existing feature request for this?

- [X] I have searched the existing feature requests

### 文档

https://kaiyuanshe.feishu.cn/wiki/VpY9wRitDiiObVkNsXycWP3Gnmf#share-Gm2UdTa6MofiVkxqS7ScRl5gnge

### 数据

https://kaiyuanshe.feishu.cn/wiki/CWsuwaIlsikquYkngAccoWCxndf?table=tblrlnv72VKbqAXd&view=vewZ3kf2QA

### Use Case

_No response_

### Proposed Solution

_No response_

### LINKED ISSUE #330: 增加“提案库”页面
### Is there an existing feature request for this?

- [X] I have searched the existing feature requests

## 文档

https://kaiyuanshe.feishu.cn/wiki/VpY9wRitDiiObVkNsXycWP3Gnmf#share-Gm2UdTa6MofiVkxqS7ScRl5gnge

## 数据

https://kaiyuanshe.feishu.cn/wiki/CWsuwaIlsikquYkngAccoWCxndf?table=tblQ08FpkMXB70JE&view=vew3Sn9qqr

### Use Case

https://www.openeuler.org/zh/oEEP/?name=oEEP-0000%20oEEP%20%20%E7%B4%A2%E5%BC%95

### Proposed Solution

_No response_

### PR DESCRIPTION
**Pull request by luojiyin1987**
[PR-371](https://github.dev/kaiyuanshe/kaiyuanshe.github.io/pull/371) [PR-371](https://codespaces.new/kaiyuanshe/kaiyuanshe.github.io/pull/371) [PR-371](https://gitpod.io/?autostart=true#https://github.com/kaiyuanshe/kaiyuanshe.github.io/pull/371) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=kaiyuanshe&utm_campaign=badge_info)

Checklist（清单）:

- [x] Labels
- [x] Assignees
- [x] Reviewers

Closes #329
Closes #330

### CODE REVIEW

File: components/Proposal/Card.tsx [hunk 1]
```diff
@@ -14,21 +16,66 @@ export const ProposalCard: FC<ProposalCardProps> = ({
...
+              </a>
+            </Row>
+          ))}
+      </details>
     </Card.Body>
-    <Card.Footer className="d-flex flex-wrap justify-content-between align-items-center">
-      <div>{createdBy as string}</div>
-      <div>{meetings as string}</div>
-    </Card.Footer>
+    <Card.Footer className="d-flex flex-wrap justify-content-between align-items-center"></Card.Footer>
```
Comment (TechQuery) [inline components/Proposal/Card.tsx:79]: 此处可以显示时间戳之类的单项内容。

File: translation/en-US.ts [hunk 2]
```diff
@@ -263,7 +264,7 @@ export default {
   // Issue pages
   issue_box: 'Issue Box',
   submit_issue: 'Submit New Issue',
-
+  detail: 'Detail',
   // Proposal pages
```
Comment (TechQuery) [inline translation/en-US.ts:268]:
```suggestion
  detail: 'Detail',

   // Proposal pages
```

**Review by TechQuery — CHANGES_REQUESTED** [review]
多个地方的同类问题请一起改。

File: components/Issue/Card.tsx [hunk 3]
```diff
@@ -44,6 +59,40 @@ export const IssueCard: FC<IssueCardProps> = ({
...
+        {Array.isArray(proposals) &&
+          (proposals[0] as TableCellRelation).text_arr.map((text, index) => (
+            <Row key={index} className="mt-2">
+              <Col>{text}</Col>
+            </Row>
+          ))}
+      </details>
+      <details>
+        <summary>{t('detail')}</summary>
+        {detail && marked(detail as string)}
```
[the remark this answers is not in the document]
Comment (luojiyin1987) [reply components/Issue/Card.tsx:94]: 是 `dangerouslySetInnerHTML`  吧 ？

[the remark this answers is not in the document]
Comment (TechQuery) [reply components/Issue/Card.tsx:94]: > 是 `dangerouslySetInnerHTML`  吧 ？ 

嗯，太长记不住。

File: components/Issue/Card.tsx [hunk 4]
```diff
@@ -44,6 +46,59 @@ export const IssueCard: FC<IssueCardProps> = ({
           )}
         </Col>
       </Row>
+      <Row>
+        <Col className="text-decoration-none align-self-end" xs={10}>
+          {textJoin(t('related'), t('department'))}: {department as string}
+        </Col>
+      </Row>
```
Comment (TechQuery) [inline components/Issue/Card.tsx:53]:
```suggestion
      {textJoin(t('related'), t('department'))}: 
      <a className="text-decoration-none" href={`/department/${department}`}>
          {department as string}
      </a>
```

File: components/Issue/Card.tsx [hunk 5]
```diff
@@ -44,6 +46,59 @@ export const IssueCard: FC<IssueCardProps> = ({
...
+      <details>
+        <summary>{textJoin(t('related'), t('meeting'))}</summary>
+        <ol>
+          {Array.isArray(meetings) &&
+            (meetings[0] as TableCellRelation).text_arr.map((text, index) => (
+              <li key={index} className="mt-2">
```
Comment (TechQuery) [inline components/Issue/Card.tsx:59]:
```suggestion
            (meetings[0] as TableCellRelation).text_arr.map((text, index) => (
              <li key={text} className="mt-2">
```

**Review by TechQuery — CHANGES_REQUESTED** [review]

**Review by TechQuery — APPROVED** [review]

**Pull request merged**